### DISCUSSION TITLE
Neo4j Package  (pip install neo4j)

### DISCUSSION THREAD

**Discussion by PreetamGKatakali** [langflow-ai/langflow#5053] [Q&A]
Hi , i just wanted to ask i am using the neo4j package (from langchain_community.graphs import Neo4jGraph) when i build the component it is giving me the pip install neo4j where can i do that ?

**Comment by anovazzi1** [comment]
are you using the cloud version or the self-hosted one?

**Reply by qdrddr** [reply]
Self hosted
